item_purchase_nevermore.lua

```diff
@@ -6,6 +6,7 @@ local ItemPurchaseSystem = dofile(GetScriptDirectory() .. "/util/ItemPurchaseSys
 
 local ItemsToBuy =
 {
+	"item_tango",

```

Comment:
Consumables don't seem to need to be added to each hero's buy list, it should be automatically bought in the public item purchasing module.

Reply:
It was discovered that if the consumables were not added to the hero's buy list the hero will not buy the said consumables. Perhaps the public item purchasing module did not work properly?

Reply:
Maybe, we could fix it later.